x/txfees/keeper/mempool-1559/code.go

```diff
@@ -151,7 +151,10 @@ func (e *EipState) updateBaseFee(height int64) {
 	if height != e.lastBlockHeight {
 		fmt.Println("Something is off here? height != e.lastBlockHeight", height, e.lastBlockHeight)
 	}
-	e.lastBlockHeight = height
+
```

Comment:
note the comment above this, updateBaseFee is called at the end of every block, so this is after all transactions have been submitted, therefore setting the last blockHeight to the next block makes sense.